timeseries/setup.py

```diff
@@ -29,7 +29,7 @@
     "lightning",  # version range defined in `core/_setup_utils.py`
     "pytorch_lightning",  # version range defined in `core/_setup_utils.py`
     "statsmodels>=0.13.0,<0.15",
-    "gluonts>=0.13.1,<0.14",
+    "gluonts==0.14.0rc2",
```

Comment:
Did we do a quick benchmark to ensure no performance regressions?

Reply:
Haven't gotten to running the benchmark yet. If we notice any regressions, we can try to address them in v0.14.1.